sdk/sdk.yaml

```diff
@@ -2934,7 +2934,7 @@ paths:
         content:
           application/json:
             schema:
-              $ref: ../evm_body.yaml#/debug_traceCall
+              $ref: ./evm_body.yaml#/debug_traceCall
```

Comment:
the reason why we have ../ is for the github actions commands to work. It will not work locally but will allow GH actions to work which is what we want! sorry it's a bit confusing but for testing u can have ./ but when you submit the PR it should be ../

can u fix this?